packages/api-demo/package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "containerjs-api-bundle": "0.0.2",
     "containerjs-api-electron": "0.0.2",
+    "containerjs-compatibility-api": "0.0.1",
```

Comment:
I only included this as a simple demo for how it works. Do you want to remove the demo, or add a new demo in the `containerjs-compatibility-api` project?